src/pages/Projects.jsx

```diff
@@ -10,147 +10,179 @@ import Grid from '@mui/material/Grid';
+            {title}
           </Typography>
+        )}
+        {status && (
           <Chip
-            label={status ?? null}
-            color="quaternary"
+            label={status}
             variant="outlined"
-            sx={{ backgroundColor: '#DEDEDEB2', fontWeight: 'bold' }}
+            sx={{ backgroundColor: '#DEDEDEB2', fontWeight: 'bold', minWidth: '120px' }}
           />
+        )}
+      </Stack>
+
+      <Divider />
+
+      {description && (
+        <Stack textAlign={{ xs: 'center', sm: 'left' }} my={2} spacing={2}>
+          <Typography variant="h6" component="h4" sx={{ fontWeight: 'bold' }}>
+            Overview
+          </Typography>
+          <Typography variant="body1" pl={{ xs: null, sm: 2 }}>
+            {description}
+          </Typography>
         </Stack>
-        {/* Title and status end */}
-        <Divider />
-        {/* Content */}
-        <Typography
-          variant="h6"
-          component="h4"
-          sx={{
-            fontWeight: 'bold',
-            mb: 2,
-            mt: 3,
-            textAlign: { xs: 'center', sm: 'left' }
-          }}
-        >
-          Overview
-        </Typography>
-        <Typography variant="body1">{description ?? null}</Typography>
-        <Typography
-          variant="h6"
-          component="h4"
-          sx={{
-            fontWeight: 'bold',
-            my: 2,
-            textAlign: { xs: 'center', sm: 'left' }
-          }}
-        >
-          Technology Used
-        </Typography>
-        <Typography variant="body1" sx={{ textAlign: 'justify' }}>
-          {techStack ?? null}
-        </Typography>
-        {/* Content ends */}
-      </CardContent>
-    </Grid>
-  );
-};
+      )}
+      {techStack && (
+        <Stack textAlign={{ xs: 'center', sm: 'left' }} my={2} spacing={2}>
+          <Typography variant="h6" component="h4" sx={{ fontWeight: 'bold' }}>
+            Technology Used
+          </Typography>
+          {/* <Stack
+            direction="row"
+            justifyContent={{ xs: 'center', sm: 'left' }}
+            spacing={{ xs: 1, sm: 2 }}
+          >
+            {techStack.map((tech) => (
+              <Typography
+                key={tech}
+                variant="body1"
+                sx={{
+                  backgroundColor: 'primary.main',
+                  borderRadius: '25px',
+                  paddingX: '10px',
+                  color: 'white'
+                }}
+              >
+                {tech}
+              </Typography>
+            ))}
+          </Stack> */}
+          <Grid
+            container
+            spacing={{ xs: 1, sm: 2 }}
+            justifyContent={{ xs: 'center', sm: 'flex-start' }}
+          >
+            {techStack.map((tech) => (
+              <Grid item key={tech}>
+                <Typography
+                  variant="body1"
+                  sx={{
+                    backgroundColor: 'primary.main',
+                    borderRadius: '25px',
```

Comment:
Does `borderRadius` actually go to `25px`? For me I only notice a difference up to `15px`.

Reply:
Honestly, I have no idea. I just always choose something high so it’s a half circle on end.  I just read all of mdn documentation and couldn’t find anything on this either. 

No issues with changing it, I just seem to always guess if Im using a px value - as I don’t understand what’s the max.

Reply:
switched to `15px`